### ISSUE TITLE
Question: root logger taking over is expected?

### ISSUE THREAD

**Issue by Lauriy** [PaulGilmartin/django-queryhunter#1]
![image](https://github.com/PaulGilmartin/django-queryhunter/assets/592465/03b4ffdd-f915-4659-ae44-573ddce15466)

```
QUERYHUNTER_BASE_DIR = queryhunter.default_base_dir(__file__)
QUERYHUNTER_REPORTING_OPTIONS = queryhunter.LoggingOptions()

MIDDLEWARE += ['queryhunter.middleware.QueryHunterMiddleware', ]
```

```
LOGGING = {
    ...
    'loggers': {
        ...
        '': {
            'handlers': ['file', 'console', 'graylog'],
            'level': 'DEBUG',
            'propagate': True
        }
    }
}
```

I get that you can't be responsible for everyone's little project, I just noticed I get nothing in queryhunter.log if I follow the simplest setup.

**Comment by PaulGilmartin** [comment]
@Lauriy Thank you for raising this :). I'm not too sure why that should happen, I can try investigating a bit later. Probably I didn't fully test this when loggers are already configured in settings.py.

What process are you running to invoke queryhunter middleware? Are you doing it via a unit test or an actual API request?

**Comment by Lauriy** [comment]
Django's internal server run, Pycharm does it for me:
`/home/lauri/PycharmProjects/eregcms/venv/bin/python /home/lauri/PycharmProjects/eregcms/manage.py runserver 127.0.0.1:8000 --settings=settings.local`

local.py does `from settings.includes.base import *` and that's where the LOGGING resides.

**Comment by PaulGilmartin** [comment]
Thanks for the info - I'll try to replicate your setup over the weekend and get a fix in as soon as possible

**Comment by PaulGilmartin** [comment]
@Lauriy I have released a fix for this in v0.1.4. Please see the updated [logging options](https://github.com/PaulGilmartin/django-queryhunter/commit/b61c568e5d210f6faa8564419f89fd33a66d79cd) docs to see how to use the updated version. Let me know if this resolves your issue :)
